pgxpool/doc.go

```diff
@@ -4,13 +4,12 @@ pgxpool implements a nearly identical interface to pgx connections.
 
 Creating a Pool
 
-The primary way of creating a pool is with `pgxpool.New`.
+The primary way of creating a pool is with [pgxpool.New]:
 
     pool, err := pgxpool.New(context.Background(), os.Getenv("DATABASE_URL"))
 
 The database connection string can be in URL or DSN format. PostgreSQL settings, pgx settings, and pool settings can be
-specified here. In addition, a config struct can be created by `ParseConfig` and modified before establishing the
-connection with `ConnectConfig`.
+specified here. In addition, a config struct can be created by [ParseConfig].
```

Comment:
There is no `ConnectConfig` in `pgxpool`, so I removed it there and in other places in this package